src/com/lilithsthrone/game/character/npc/dominion/Brax.java

```diff
@@ -503,7 +509,8 @@ public String getContent() {
 					+ "</p>"
 					+ "<p>"
 						+ "Taking one last look around Brax's office, you notice that there's a neatly-folded spare uniform lying on top of a cabinet."
-						+ " Deciding to punish Brax a little more for trying to beat you up, <b>you take the spare uniform, and add it to your inventory</b>."
+						+ " It also has a scroll sitting on top of the pile of uniform pieces."
```

Comment:
This sentence feels clunky to me. Maybe something like "...there's a neatly-folded spare uniform lying on top of a cabinet, next to a small scroll.